File: 백준/Bronze/4892. 숫자 맞추기 게임/숫자 맞추기 게임.py
import sys
input = sys.stdin.readline

N = int(input())

cnt = 1
func = lambda x:"odd" if (x%2==1) else "even"

while N != 0 :
    print(cnt, ". ", func(N), " ", N//2, sep="")
    N = int(input())
    cnt += 1

File: 백준/Bronze/4892. 숫자 맞추기 게임/숫자 맞추기 게임.java
import java.io.*;

public class Main {
	public static void main(String[] args) throws NumberFormatException, IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();

        int cnt = 1;
    	int N = Integer.parseInt(br.readLine());

        while (N != 0) {
        	sb.append(cnt++).append(". ").append(N%2==0?"even":"odd").append(" ").append(N/2).append("\n");
        	N = Integer.parseInt(br.readLine());
		}
        
        System.out.println(sb);
        br.close();
    }
}